client/app/vps/translations/Messages_en_GB.xml

```diff
@@ -43,6 +43,10 @@
    <translation id="vps_monitoring_cpu_sys" qtlid="83733">System</translation>
    <translation id="vps_monitoring_cpu_user" qtlid="7450">User</translation>
 
+   <translation id="vps_dashboard_upgrade_1">An update of the infrastructure VPS 2016 & 2018 in area os-gra1-005, os-gra1-006, os-gra1-007, os-gra1-008, os-gra1-009, os-gra1-010 and os-gra1-011 is scheduled on Wednesday 4th July from 9 AM UTC+2 to 5 PM UTC+2.</translation>
```

Comment:
I know we're not supposed to do it, there is not other way to be able to prod today